utilities/Utils.ts

```diff
@@ -28,6 +30,16 @@ export class Utils {
         return list[Math.floor(Math.random() * list.length)];
     }
 
+    /**
+     * Get random user model
+     * @returns a random user model
+     */
+    public static getRandomUserModel(): UserModel {
+        const user = new UserModel();
+        user.updateStats(321, 2321, 345, 435, 324, 2432)
+        return user;
+    }
+
```

Comment:
Is this function for testing purposes only ? You should add a TODO : remove after testing if so